rqt_top/src/rqt_top/top_plugin.py

```diff
@@ -34,6 +34,8 @@
 from python_qt_binding.QtCore import Qt, QTimer
 
 from rqt_top.node_info import NodeInfo
+from rqt_top.msg import *
```

Comment:
I agree with this sentiment. one should never import * it just makes things impossible to read.